tensorboard/data/experimental/experiment_from_dev_test.py

```diff
@@ -0,0 +1,206 @@
+                            else:
+                                value = -1.0 / (step + 1)
+                                seconds = 600 + step
+                        else:  # "accuracy"
+                            if run == "train":
+                                value = 1.0 / (10 - step)
+                                seconds = step * 2
+                            else:
+                                value = -1.0 / (10 - step)
+                                seconds = 600 + step * 2
+                        response.points.values.append(value)
+                        response.points.wall_times.add(seconds=seconds, nanos=0)
+                    yield response
+
+        mock_api_client.StreamExperimentData = mock.Mock(
+            wraps=stream_experiment_data
+        )
+
+        with mock.patch.object(
+            experiment_from_dev,
+            "get_api_client",
+            lambda api_endpoint: mock_api_client,
+        ):
+            experiment = experiment_from_dev.ExperimentFromDev("789")
+            for pivot in (None, False):
+                with self.subTest("pivot=%s" % pivot):
+                    dataframe = experiment.get_scalars(pivot=pivot)
+
+                    expected = pandas.DataFrame(
+                        {
+                            "run": ["train"] * 20 + ["test"] * 20,
+                            "tag": (["accuracy"] * 10 + ["loss"] * 10) * 2,
+                            "step": list(np.arange(0, 10)) * 4,
+                            "wall_time": np.concatenate(
+                                [
+                                    2.0 * np.arange(0, 10),
+                                    1.0 * np.arange(0, 10),
+                                    600.0 + 2.0 * np.arange(0, 10),
+                                    600.0 + np.arange(0, 10),
+                                ]
+                            ),
+                            "value": np.concatenate(
+                                [
+                                    1.0 / (10.0 - np.arange(0, 10)),
+                                    1.0 / (1.0 + np.arange(0, 10)),
+                                    -1.0 / (10.0 - np.arange(0, 10)),
+                                    -1.0 / (1.0 + np.arange(0, 10)),
+                                ]
+                            ),
+                        }
+                    )
+
+                    if pivot is None:  # Default behavior: pivot_table.
+                        pandas.testing.assert_frame_equal(
+                            dataframe,
+                            expected.pivot_table(
+                                ["value", "wall_time"], ["run", "step"], "tag"
+                            ),
+                            check_names=True,
+                        )
+                    else:  # pivot == False
+                        pandas.testing.assert_frame_equal(
+                            dataframe, expected, check_names=True
```

Comment:
fwiw, `check_names=True` is the default.

Reply:
Yes, I'm aware of that. I prefer to be explicit and safer against potential future changes.